### ISSUE TITLE
Trinket slots

### ISSUE THREAD

**Issue by p3lim** [Strubbelkopp/Quiver#2]
I saw you've got plans to add Trinkets support, and while I'm not going to nag about that specifically, I'm just going to suggest that when you do add support for it it'd support not just having the quiver on the back. Quivers can be equipped on the hip as well (belt slot in Trinkets), leaving the already crowded back slot in Trinkets free for stuff like backpacks and elytra for those who want that.

**Comment by Strubbelkopp** [comment]
Yeah, thought about doing that. Don't have much time at the moment, but I'll do it once I get around to it
